README.rst

```diff
@@ -90,6 +90,25 @@ And here is a sparse adjacency matrix:
     from scipy.sparse import random
     adj_sparse = random(100, 100, density=0.1)
     dgm = ripser_parallel(adj_sparse, metric="precomputed", maxdim=2, n_threads=-1)
+
+Edge Collapser
+--------------
+
+Push the computation to higher homology dimensions and larger point clouds/distance matrices/adjacency matrices using edge collapses:
```

Comment:
This text was there before, I just moved it below. I can't think of a short alternative that is immune to the risk you point out. Do you have ideas?

Reply:
"Can unlock the exploration of higher dimensions and/or process bigger data-set at the cost of some pre-processing time" but even here I am not 100% satisfied. I think that in the paper the table showing the results with different data-sets allows user to better understand when to use EC.

Reply:
Right, but does all this belong to a basic usage section? Personally, I thought the point was to show code snippets that illustrate the capabilities in a quick way.

Reply:
Furthermore, the "pre-processing" is not exposed to the user, because they don't get to see the input after pre-processing by EC.

Reply:
I think the note is perfect, and maybe refer users to the paper in case they want more details ?

BTW, you raise a good point :
> Furthermore, the "pre-processing" is not exposed to the user, because they don't get to see the input after pre-processing by EC.

Do you think that users could be interested about the collapsed graph in the future ?

Reply:
Added the note.

> Do you think that users could be interested about the collapsed graph in the future ?
Yes, I think in the future it would be good to expose the edge collapse function to users.

Reply:
LGTM